docs/user-guide/settings/advanced.md

```diff
@@ -1,92 +1,161 @@
 # Advanced Settings #
 
-## Options ##
+This page enables advanced control of PhotoPrism,
+and configuring image and thumbnail handling.
 
-#### Debug Logs ####
+![](img/advanced-settings.jpg)
+
+The same options can all be controlled instead using their corresponding
+[config options](/getting-started/config-options/).
+Settings updated here will override values already configured.
+
+!!! note
+    The Advanced Settings page is not available if public mode is [enabled](/getting-started/config-options/).
+
+## Options
+
+### Debug Logs
 When enabled, debug logs are shown in *Library>Logs*.
-Requires re-start.
+Requires restart.
+
+The equivalent [config toggle](/getting-started/config-options/) is `PHOTOPRISM_DEBUG`.
 
-#### Read-only Mode ####
+### Read-only Mode
 When enabled, importing, uploading and deleting files is not possible.
 
-#### Experimental Features ####
+The equivalent [config toggle](/getting-started/config-options/) is `PHOTOPRISM_READONLY`.
+
+### Experimental Features
 When enabled, your instance will be updated with experimental features.
 
-#### Disable Backups ####
-This option prevents PhotoPrism from creating the following yml backups:
+The equivalent [config toggle](/getting-started/config-options/) is `PHOTOPRISM_EXPERIMENTAL`.
+
+### Disable Backups
+This option prevents creating the following `yml` backups:
 
 - For photos in `storage/sidecar`
 - For albums, months, states and folders in `storage/albums`
 
-#### Disable WebDAV ####
-This option prevents building WebDav connections. A restart is required for changes to be applied.
+The equivalent [config toggle](/getting-started/config-options/) is `PHOTOPRISM_DISABLE_BACKUPS`.
 
-#### Disable Places ####
-When selected, no geo-information is read from your files metadata. The Places section will not be visible.
+### Disable WebDAV
+This option prevents building WebDav connections.
+Requires restart for changes to be applied.
 
-#### Disbale ExifTool ####
-This option prevents the creation of json files with exif data in `storage/sidecar`. 
+The equivalent [config toggle](/getting-started/config-options/) is `PHOTOPRISM_DISABLE_WEBDAV`.
 
-#### Disable TensorFlow ####
-When selected, no labels are created for your files.
+### Disable Places
+When selected, no geo-information is read from your files metadata.
```

Comment:
Lat/long will be indexed, but not resolved via reverse lookups to locations (country, state, city, location type,...)

Reply:
Brilliant, thanks. I'll update to clarify this.